src/Pipeline.h

```diff
@@ -88,16 +117,25 @@ struct CustomLoweringPass {
 struct JITExtern;
 
 struct AutoSchedulerResults {
-    std::string scheduler_name;          // name of the autoscheduler used
-    Target target;                       // Target specified to the autoscheduler
-    std::string machine_params_string;   // MachineParams specified to the autoscheduler (in string form)
+#ifdef HALIDE_ALLOW_LEGACY_AUTOSCHEDULER_API
```

Comment:
Does setting this ifdef trigger a deprecation warning?

Reply:
Nope. But I guess it should. Will fix.